python/cugraph/cugraph/structure/graph_classes.py

```diff
@@ -282,13 +282,8 @@ def from_dask_cudf_edgelist(
             If True, stores the transpose of the adjacency matrix.  Required
             for certain algorithms.
 
-        legacy_renum_only : bool, optional (default=False)
```

Comment:
You probably shouldn't remove the description if the arg is still available to be used.  You should instead mention that it's deprecated and/or explain how it's supposed to be used.  If it's being deprecated here, then we should also be raising a warning.

Also, shouldn't we be updating `from_cudf_edgelist()` too?

Reply:
Right. I believe this was resolved.